docs/deploy.rst

```diff
@@ -13,16 +13,18 @@ pairs, exodus-storage.yaml, run by the `AWS CLI`_ command
 ``aws cloudformation deploy``.
 
 - Parameters
-    - env
+    - env:
         The environment in which to deploy functions.
         Available values: dev, stage, prod
         Default: dev
-    - project
+    - project:
         The project name under which the resources are created.
         Default: exodus
+    - oai:
+        The origin access identity ID associated with the environment.
 
-Additionally, an `Origin Access Identity`_ is created for use with
-`Amazon CloudFront`_ distributions.
+An `Origin Access Identity`_ is created for use with
+`Amazon CloudFront`_ distributions, should be created in advance.
```

Comment:
This text doesn't seem to make sense as it says an OAI "is created for use", i.e. these commands will create the OAI, but then "should be created in advance", i.e. it's your responsibility to create the OAI before running these commands.  Which is it?  Maybe this sentence should just be removed?

Reply:
Yeah, I actually agree that this whole sentence should just be removed.